Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView+SwiftUI.swift

```diff
@@ -9,6 +9,7 @@ import SwiftUI
 @available(iOS 13.0, *)
 /// Protocol of `ChatChannelListItemView` wrapper for use in SwiftUI.
 public protocol ChatChannelListItemViewSwiftUIView: View {
+    @available(*, deprecated, message: "This is now deprecated, please refer to the SwiftUI SDK at https://github.com/GetStream/stream-chat-swiftui")
```

Comment:
The deprecation flags should actually be above the ChatChannelListItemViewSwiftUIView. Any reason why they are on the initialisers?